stable/artifactory-ha/templates/_helpers.tpl

```diff
@@ -10,14 +10,32 @@ Expand the name of the chart.
 The primary node name
 */}}
 {{- define "artifactory-ha.primary.name" -}}
-{{- default "artifactory-ha-primary" .Values.artifactory.primary.name | trunc 63 | trimSuffix "-" -}}
+{{- if .Values.artifactory.primary.nameOverride -}}
```

Comment:
I think you mixed up the `name` templates with the `fullname` templates. Revert that change. The previous version was fine.

Reply:
I reverted, but kept the unique naming. It's very useful.